### ISSUE TITLE
How to make them work?

### ISSUE THREAD

**Issue by DearFox** [McTsts/shader-utilities#5]
I was curious to try some of these, but some of them I don't understand how to enable.
i created a resource pack and created a folder \assets\minecraft\shaders
Should I throw it right at her? Or maybe I should create a core folder and put it in it like it is in other shaders?

![image](https://user-images.githubusercontent.com/59219907/166452472-4bf63d5a-63fd-4bb6-bf10-e316f1674708.png)

**Comment by McTsts** [comment]
for this one the shaders need to be in shaders/core (as they are in the image), icons.png replaces the vanilla icons.png file which is located in textures/gui

**Comment by soshimee** [comment]
I'm confused as to where I put the offset in, I think it's in main but it doesn't really change anything. I already have a custom icons.png (but the hearts are unmodified)

**Comment by McTsts** [comment]
Im not sure what you mean

**Comment by soshimee** [comment]
TLDR IDK how to use this.

**Comment by McTsts** [comment]
Put the shaders into shaders/core and icons.png replaces the vanilla icons.png file which is located in textures/gui

**Comment by soshimee** [comment]
What do you mean by icons.png replaces vanilla icons.png? I don't use the vanilla one, my resource pack already has a custom one.

**Comment by McTsts** [comment]
Then you'll need to merge them

**Comment by McTsts** [comment]
The hearts have a special level of opacity so the shader can target them

**Comment by soshimee** [comment]
Oh okay, let me see.

**Comment by soshimee** [comment]
It works! Can I move other elements as well? Maybe give them different alpha? For some reason, it looks like the hearts have 1 less RGB value than the original but same alpha as the original, why is that?

**Comment by soshimee** [comment]
Oh I see, it's alpha in the background invisible pixels!

**Comment by soshimee** [comment]
Then why not set the RGB value in those instead of alpha? Then it won't be visible even with hundreds of adjustments (although no one would do that)

**Comment by McTsts** [comment]
unsure what you mean

**Comment by soshimee** [comment]
So like put RGBA (0, 0, 0, 1) for health and RGBA (1, 0, 0, 1) for hunger.

**Comment by soshimee** [comment]
Would there be a way to shift the hearts in a way so it disappears in the other direction? Maybe mirror it in the icons.png then re-mirror the entire thing? But not sure how...
